docs-source/content/samples/simple/domains/model-in-image/_index.md

```diff
@@ -1717,8 +1749,890 @@ Here are the steps:
+
+        The model file in the image uses macro `@@ENV:CUSTOM_DOMAIN_NAME@@` to reference this environment variable when setting its domain name. 
+
+        Specifically, change the corresponding domain resource `spec.serverPod.env` YAML stanza to look something like this:
+
+        ```
+        ...
+        spec:
+          ...
+          serverPod:
+          ...
+            env:
+            - name: CUSTOM_DOMAIN_NAME
+              value: "domain2"
+          ...
+        ```
+
+      - Change the `/tmp/mii-sample/mii-update2.yaml` domain resource `spec.domainHome` value to `/u01/domains/sample-domain2`. The corresponding YAML stanza should look something like this:
+
+        ```
+        ...
+        spec:
+          ...
+          domainHome: /u01/domains/sample-domain2
+          ...
+        ```
+
+        (This change is not strictly needed, but it is a helpful convention to decorate a WebLogic domain's home directory with its domain name or domain UID.)
+
+      - Change the `/tmp/mii-sample/mii-update2.yaml` secret references in the `spec.webLogicCredentialsSecret` and `spec.configuration.secrets` stanzas to reference this use case's secrets. Specifically, change:
+
+          ```
+          spec:
+            ...
+            webLogicCredentialsSecret:
+              name: sample-domain1-weblogic-credentials
+            ...
+            configuration:
+            ...
+              secrets:
+              - sample-domain1-datasource-secret
+              ...
+              model:
+                ...
+                runtimeEncryptionSecret: sample-domain1-runtime-encryption-secret
+          ```
+
+        To this:
+
+
+          ```
+          spec:
+            ...
+            webLogicCredentialsSecret:
+              name: sample-domain2-weblogic-credentials
+            ...
+            configuration:
+              ...
+              secrets:
+              - sample-domain2-datasource-secret
+              ...
+              model:
+                ...
+                runtimeEncryptionSecret: sample-domain2-runtime-encryption-secret
+          ```
+
+        > __NOTE__: If you are following the `JRF` path through the sample, similarly change your `spec.configuration.opss.walletPasswordSecret` and the RCU secret name referenced in `spec.configuration.secrets`.  
+
+
+      - Change the domain resource's `spec.configuration.model.configMap` value from `sample-domain1-wdt-config-map` to `sample-domain2-wdt-config-map`. The corresponding YAML stanza should look something like this:
+         ```
+         spec:
+           ...
+           configuration:
+             ...
+             model:
+             ...
+               configMap: configMap: sample-domain2-wdt-config-map
+         ```
+
+      - Now, let's compare your original and changed domain resource files to double check your changes.
+
+        {{%expand "Click here to see the expected 'diff'." %}}
```

Comment:
this seems material, i would not hide in an expander

Reply:
done